src/js/app.js

```diff
@@ -1 +1,126 @@
 import '../css/app.scss';
+import { convertTemperature, getCurrentWeather } from './api-client';
+
+
+/**
+ * We have to disable arrow functions with prototype modifications
+ * because we need the this object inside the functions.
+ */
+
+/**
+ * Clears a node after n seconds
+ */
+// eslint-disable-next-line func-names
+Node.prototype.clearAfter = function (seconds) {
```

Comment:
For this eslint rule above wouldn't adding a name after the function keyword solve the problem :thinking: I believe it would